packages/web/components/denom/denom-image.tsx

```diff
@@ -0,0 +1,29 @@
+import { FunctionComponent } from "react";
+import Image from "next/image";
+
+import { DenomImageProps } from "./types";
```

Comment:
Minor: Could we move this into assets/ folder. That's where we've had asset-related components. Yes, I'm aware it clashes with assets from a web dev perspective (images, icons, etc.)